### DISCUSSION TITLE
習慣的養成

### DISCUSSION THREAD

**Discussion by GuanYann** [1101-GS4719-C/Python-Community-GS4719-C#1217] [General]
好的習慣是需要長時間的培養和持續的實踐，而不良習慣可能在不知不覺中形成。一旦習慣成形，大腦會自動化這些行為模式，使得改變變得困難。因此，重要的是要時常反思自己的習慣，並刻意改變那些不好的習慣，才能建立更健康和有益的生活模式。好習慣對成長與發展很重要，可提高效率、時間管理和自律能力。因此，我們應從小事開始培養好習慣，最終造就更好的自己。

https://web.cheers.com.tw/issue/2020/21habits/article/01.html

**Comment by Hsucuchia** [comment]
好的習慣對於個人成長和發展極具重要性，建立健康和有益的生活模式需要長時間的努力和持續的實踐。同時，它也提醒了我們不良習慣可能會在不知不覺中形成，因此我們需要時常檢視自己的習慣並刻意改變那些不好的習慣。通過培養好習慣，我們可以提高效率、時間管理和自律能力，進而實現自我成長和發展。

**Comment by KevinCheng42** [comment]
培養良好習慣有助於長遠的發展，我覺得可以效仿《原子習慣》此書中所提到的，透過讓提示顯而易見，如將慢跑鞋擺在座位旁，隨時提醒自己要運動；抑或是讓行動輕而易舉，如把要讀的書隨時擺在書桌上，提高好習慣養成的可能性。另一方面，若要擺脫壞習慣，可以藉由與培養好習慣相反的行為如讓提示消失或是讓行動難以達成，以破除壞習慣～

**Comment by DuxtinTai** [comment]
同意你的觀點，透過培養良好的習慣讓自己的生活更有規律性和效率性，《原子習慣》中提到的方法，對我來說也非常實用。比如說，我之前曾經有過拖延症，總是把事情留到最後一刻才去完成。後來我就嘗試將需要完成的任務寫在桌曆上，並將桌曆在顯眼的地方，提醒自己不要拖延。另外，我也會把需要完成的事情分成小步驟，逐步完成，這樣不僅可以減輕心理負擔，也能增強自信心。

**Comment by Evening218** [comment]
我覺得要培養習慣最重要的是想清楚自己的動機,或是找到一個足以支持自己的動力。常常聽到一個說法是:21天就可以培養一個習慣,但事實是如果自己沒有下定決心,是很難成功培養出好的習慣的。以我來說,運動無庸置疑是個值得培養的好習慣,但我嘗試無數次都沒堅持超過一周(哭),所以我認為應該先努力尋找可以說服自己培養好習慣的理由。

**Comment by tower20021202** [comment]
我覺得比起刻意培養一個好的習慣，更重要的是應該要找個強烈的動機讓你可議維持著這些習慣，不然刻意培養的習慣往往很容易輕易的被打破

**Comment by ziihui** [comment]
好的習慣對我們超重要的，可以讓我們更有效率、管理時間和自律，而且好習慣養成真的需要花很多時間。反之，壞習慣很容易不知不覺就培養成，對我們的生活造成負面的影響。所以要時常反省自己的習慣，並盡力改掉那些不好的部分。從一些小事情開始，培養好習慣，讓自己成為更好的人。

**Comment by Jack-1D** [comment]
之前也曾經有老師跟我們說過，大家都經常聽到"習慣成自然"這句話，卻忽略了前一句話"勉強成習慣"。我認為一個習慣的養成是需要下苦心的。而好的習慣也能在人生中的某個階段對我們產生幫助。

**Comment by Chiuhsianglin** [comment]
很常在網路上會看到有人說只要堅持21天就可以養成一個習慣，但去年暑假我因為要打工有一個月都五點半起床，但這個月過後並沒有讓我養成早起的習慣🤣

**Comment by YoYooHuang** [comment]
我覺得習慣真的很重要，好的習慣能決定一個人的生產力以及做事的效率。而關於習慣養成的技巧，我推薦“原子習慣”這本書，裡面敘述了許多關於習慣養成的好處，也帶著讀者用最簡單最生活化的技巧達到養成習慣的目的！

**Comment by Jamesmoneya** [comment]
好的習慣會讓你的生活多采多姿，壞的習慣讓你一敗塗地，首要先將你的壞習慣一一挑出，一次一次的省察，透過對你最好的方式徹底地將壞習慣剷除，例如經常熬夜，關燈看手機，邊走路邊划手機，這些小小的壞習慣，很有可能讓你暴斃失明車禍，因此快點改掉你的壞習慣吧

**Comment by hardtagottaname** [comment]
習慣真的是一個很可怕卻又很重要的存在。藉由習慣，可以讓我們更加進步，卻也可能讓我們因此墮落。最近我也開始反思，應該要養成讀課外書的好習慣。讀書能增進我的內在，培養思考能力成為一個能獨立思考、不容易受別人左右想法的人。除此之外，我也應該摒除太晚睡的壞習慣，畢竟熬夜對身體免疫力和專注力都有損耗，這會導致我在上課期間無法有效吸收知識。希望我能繼續堅持下去，成功熬過專家說的30天！

**Comment by CCello** [comment]
的習慣對個人的成長和發展有著重要的影響，而不良習慣也可能會導致負面影響。持續的自我反思和刻意的改變可以幫助我們擺脫不良習慣，建立更健康和有益的生活模式。從小事情開始培養好習慣，例如早起、養成運動習慣、定期閱讀等等，這些都可以提高效率、時間管理和自律能力。這些好習慣將有助於我們在學習、工作和生活中更好地發揮自己的潛力，成為更好的自己。

**Comment by harrycoconut** [comment]
好的習慣其實並不需要長期養成，就跟壞習慣一樣，若是你做了好習慣之後有即時的回饋，那麼這項習慣在短時間內便能建立起來，以我為例，在養成早起的前三天，我每天起來都覺得精神非常好，並且一整天的產值提高快速，這之後我並沒有用什麼特殊方法，每天就會自動早起了😌

**Comment by Alexgns** [comment]
培養習慣最重要的是確定你想要培養的習慣是什麼，並設定明確的目標。目標應該是具體、可衡量和可實現的，這樣就可以更容易跟著計劃前進。

**Comment by alicia0725** [comment]
之前看到一本書說養成一個好的習慣需要21天，但對我自己而言，只要能夠做一件想養成的習慣三天，三天過後沒做到就會覺得自己半途而廢，所以會盡力做到，這樣就可以養成一個習慣了

**Comment by alstinlin** [comment]
我認為培養好習慣的好處除了提高自身效率以及讓自己更自律外，透過每天實行好習慣能夠讓我覺得生活更踏實，有持續朝著目標前進的感覺~

**Comment by PaulSu0905** [comment]
我發現好的習慣需要長期的培養和持續的練習，而壞的習慣可以在不知不覺中形成。習慣一旦形成，大腦就會將這些行為模式自動化，使改變變得困難。因此，定期反思我們的習慣，有意識地改變那些有害的習慣，以建立更健康和有益的常規是很重要的。良好的習慣對成長和發展至關重要，可以提高工作效率、時間管理和自律性。因此，我們應該從小動作開始培養良好的習慣，最終創造一個更好的自己。

**Comment by lyien1021** [comment]
我覺得要刻意養成一個習慣真的超難，之前我有嘗試在睡前看幾頁英文再睡覺，大概堅持了1個月，但因為馬上有許多考試作業出現，睡前閱讀也就不了了之了。

**Comment by larry40305** [comment]
最重要的是要耐心和有耐性，不要期望一夜之間就改變所有習慣。給自己一些寬限和容錯的空間，並不斷努力，你將逐漸建立起更健康、有益和有成效的生活模式。

**Comment by MaxChi870** [comment]
我認為習慣非一朝一夕便能養成的，需要先立定目標，與確立計劃的執行方式，及時間上的規劃，需讓自己有些許的彈性空間，讓自己每完成執行一次便有些微的成就感，才能有維持下去的動力。

**Comment by Alex11392** [comment]
好習慣的養成需要時間，但在它成為習慣之前更需要刻意的去培養，需要有意識地去限制或是強迫自己，而這也是我覺得養成好習慣的最難之處。

**Comment by ThanatosJun** [comment]
以我的經驗來看，好的習慣較難養成是因為其缺乏了誘因，對你當下生活的愉悅值沒有提升，反而下降；而壞的習慣較容易養成是因為人的惰性，我們能輕鬆就不想勞累，科技的不斷發展不就正好點明此事了嗎？人向著方便前行這件事！但有一件事情格外奇怪，那就是愛情，他彷彿能打破這項魔咒，讓人自主去培養優良的習慣，去規避大眾所認為的不優良習慣。因此若要培養一項好的習慣，與愛人一起，或為愛人，這兩著都相當有效。
